core/connect.js

```diff
@@ -4,58 +4,14 @@
 //  ENiGMA½
 const ansi          = require('./ansi_term.js');
 const Events        = require('./events.js');
+const Config        = require('./config.js').get;
 const { Errors }    = require('./enig_error.js');
 
 //  deps
 const async     = require('async');
 
 exports.connectEntry    = connectEntry;
 
-function ansiDiscoverHomePosition(client, cb) {
-    //
-    //  We want to find the home position. ANSI-BBS and most terminals
-    //  utilize 1,1 as home. However, some terminals such as ConnectBot
-    //  think of home as 0,0. If this is the case, we need to offset
-    //  our positioning to accommodate for such.
-    //
-    const done = (err) => {
```

Comment:
I think we might still want this. *If* we can query and get a response, this helps hack around 1,1 (standard) vs 0,0 (non-standard) home positions. If we can't detect it we should time out and no harm no foul.

Reply:
Good point. I'll put this back in, but maybe also put that behind a config option if you agree? That way someone who really doesn't want them can still avoid it (at the risk of some broken clients) but by default it can still do the check.

We could also theoretically combine this test with the UTF8 check (since it already does 2 stages) if it wasn't skipping that one for the "nix" clients to make only one place it does a position check still. Thoughts?

Reply:
I'm fine with putting it behind a configuration option, keeping default behavior -- that sounds great.

The problem with the UTF-8 check (kludge!) is it can bork up things with "BBS" clients. Aside: I really need to get around to cleaning up the names of all of this stuff too...

Reply:
Added it back in with a configuration option, `term.checkAnsiHomePosition`